ResidentInformationApi.Tests/V1/E2ETests/HelperMethods.cs

```diff
@@ -0,0 +1,162 @@
+using Email = ResidentInformationApi.V1.Boundary.Responses.Email;
+using HousingResidentInformation = ResidentInformationApi.V1.Boundary.Responses.HousingResidentInformation;
+using MosaicResidentInformation = ResidentInformationApi.V1.Boundary.Responses.MosaicResidentInformation;
+using Phone = ResidentInformationApi.V1.Boundary.Responses.Phone;
+using PhoneType = ResidentInformationApi.V1.Boundary.Responses.PhoneType;
+
+namespace ResidentInformationApi.Tests.V1.E2ETests
+{
+    public static class HelperMethods
+    {
+        public static string ToJson(this object toConvert)
+        {
+            return JsonConvert.SerializeObject(toConvert, new JsonSerializerSettings
+            {
+                ContractResolver = new DefaultContractResolver
+                {
+                    NamingStrategy = new CamelCaseNamingStrategy()
+                },
+                Formatting = Formatting.None,
+                Converters = new List<JsonConverter> { new Newtonsoft.Json.Converters.StringEnumConverter() }
+            });
+        }
+
+        public static void SetupMosaicResponseWithClaimants(string claimants, WireMockServer mockMosaicApi)
+        {
+            mockMosaicApi.Given(Request.Create().WithPath("/api/v1/residents").UsingGet())
+                .RespondWith(Response.Create().WithBody(claimants, encoding: Encoding.UTF8)
+                    .WithStatusCode(HttpStatusCode.OK));
+        }
+
+        public static void SetupAcademyResponseWithClaimants(string claimants, WireMockServer mockAcademyApi)
+        {
+            mockAcademyApi.Given(Request.Create().WithPath("/api/v1/claimants").UsingGet())
+                .RespondWith(Response.Create().WithBody(claimants, encoding: Encoding.UTF8)
+                    .WithStatusCode(HttpStatusCode.OK));
+        }
+
+        public static void SetupHousingResponseWithHouseholds(string households, WireMockServer mockHousingApi)
+        {
+            mockHousingApi.Given(Request.Create().WithPath("/api/v1/households").UsingGet())
+                .RespondWith(Response.Create().WithBody(households, encoding: Encoding.UTF8)
+                    .WithStatusCode(HttpStatusCode.OK));
+        }
+
+        public static IEnumerable<ResidentInformationResult> MapToResponse(HousingApiResponse households)
+        {
+            return households.Residents.Select(h => new ResidentInformationResult
+            {
+                System = "Housing",
+                SystemId = h.HouseReference,
+                SystemUrl =
+                    new Uri(
+                        $"{Environment.GetEnvironmentVariable("HOUSING_API_URL")}api/v1/households/{h.HouseReference}/people/{h.PersonNumber}"),
+                Data = new HousingResidentInformation
+                {
+                    Uprn = h.Uprn,
+                    Address = new Address
+                    {
+                        AddressLine1 = h.Address.AddressLine1,
```

Comment:
The Address object from the housing API response doesn't have `addressLines 2 & 3`.
We've got it set up to return as:
```
"address": {
    "propertyRef": value,
    "addressLine1": value,
    "postCode": value
```

Reply:
Thanks for spotting, I have now amended it :)